prettier.config.js

```diff
@@ -1,14 +1,11 @@
-/** @type {import("prettier").Config} */
-const config = {
+module.exports = {
+  printWidth: 80,
```

Comment:
Why is this modified in this PR?

Reply:
VS Code was telling me it was an invalid Prettier config, so I rewrote it to fix that.

Reply:
Typically, this should be created as a bug ticket and committed separately. We'll let it go through this time.